### PR TITLE
refactor(README): Update README for V5

### PR DESCRIPTION
**Pull request by ntilwalli**
I updated the README for V5.  I haven't actually used tested the V5 changes so this def needs a review.

### CODE REVIEW

**Comment by ntilwalli** [comment]
@jvanbruegge Ooh, you made the README update while I was doing the same!  I think we should deprecate the older README.  I also added option parameter descriptions in this one which could be pulled over to yours...

File: README.md [hunk 1]
```diff
@@ -49,22 +42,45 @@ function main(sources) {
...
-    router: xs.of('/other'),
+    router: xs.of('/other') // Notice use of 'router' sink name, 
+                            // which proxies the original 'history' sink
   };
 }
 
-Cycle.run(main, {
+const mainWithRouting = routerify(main, switchPath)
+
+Cycle.run(mainWithRouting, {
```
Comment (jvanbruegge) [inline README.md]: without `Cycle.`

**Comment by jvanbruegge** [comment]
Yeah, that was basicly just a quick&dirty fix

**Comment by ntilwalli** [comment]
Updated it

File: README.md [hunk 2]
```diff
@@ -88,22 +42,45 @@ function main(sources) {
...
   DOM: makeDOMDriver('#app'),
-  router: makeRouterDriver(createBrowserHistory(), switchPath)  // v3
-  // router: makeRouterDriver(createHistory()) // <= v2
+  history: makeHistoryDriver() // create history driver as usual,
+                               // but it gets proxied by routerify
 });
 ```
+### Routerify Options
+
+Routerify accepts an options object like so: `routerify(main, options)`
```
Comment (jvanbruegge) [inline README.md]: `routerify(main, switchPath, options)`

Comment (ntilwalli) [inline README.md]: I fixed it, directly on the README

**Pull request merged**